root/programs/C#/3_Build_NuGet_net47.bat

```diff
@@ -0,0 +1,39 @@
+setlocal
```

Comment:
@daisukenishino 

Notify developers of how to use this file in [readme](https://github.com/OpenTouryoProject/OpenTouryo/tree/develop/root#build-program).